src/main/java/org/opentripplanner/standalone/config/BuildConfig.java

```diff
@@ -373,6 +381,7 @@ public BuildConfig(JsonNode node, String source, boolean logUnusedParams) {
     maxElevationPropagationMeters = c.asInt("maxElevationPropagationMeters", 2000);
     boardingLocationTags = c.asTextSet("boardingLocationTags", Set.of("ref"));
     discardMinTransferTimes = c.asBoolean("discardMinTransferTimes", false);
+    timeZone = Optional.ofNullable(c.asText("timeZone", null)).map(ZoneId::of).orElse(null);
```

Comment:
I think the zoneId should be required, that simplifies the code - and I can not see a need to have it as an optional parameter. Also, add support for ZoneId to the NodeAdaptor. That make this code clean.

Reply:
Yes, definitely nicer this way